samples/javascript_nodejs/56.teams-file-upload/bots/teamsFileUploadBot.js

```diff
@@ -4,28 +4,31 @@
 const fs = require('fs');
 const path = require('path');
 const axios = require('axios');
-
-const {
-    TeamsActivityHandler,
-    MessageFactory
-} = require('botbuilder');
+const { TurnContext, MessageFactory, TeamsActivityHandler } = require('botbuilder');
+const { MicrosoftAppCredentials } = require('botframework-connector');
+const { GeneFileName, GetFileSize, WriteFile } = require('../services/fileService');
+const FILES_DIR = 'files';
 
 class TeamsFileUploadBot extends TeamsActivityHandler {
     constructor() {
         super();
         this.onMessage(async (context, next) => {
+            TurnContext.removeRecipientMention(context.activity);
             const attachments = context.activity.attachments;
+            const imageRegex = /image\/.*/;
             if (attachments && attachments[0] && attachments[0].contentType === 'application/vnd.microsoft.teams.file.download.info') {
                 const file = attachments[0];
                 const response = await axios.get(file.content.downloadUrl, { responseType: 'stream' });
-                const filePath = path.join('Files', file.name);
+                const filePath = path.join(FILES_DIR, file.name);
```

Comment:
I believe that in this block (I can only comment on the changed line, but I'm referencing lines 21-23), you can use your `writeFile()` service function.